misc/demo/index.html

```diff
@@ -50,6 +50,16 @@
                     </ul>
                 </li>
                 <li><a href="#getting_started">Getting started</a></li>
+                <li ng-show="latestVersion" class="dropdown" uib-dropdown>
+                    <a role="button" class="dropdown-toggle" uib-dropdown-toggle>
```

Comment:
dropdown and dropdown-toggle classes should be unnecessary - same with dropdown-menu below as well I believe